tests/infrastructure/src/main/java/org/mule/test/infrastructure/profiling/tracing/SpanTestHierarchy.java

```diff
@@ -145,6 +146,7 @@ && hasCorrectParent(exportedSpan, actualParent != null ? actualParent.getName()
     assertThat("Expected span: " + expectedNode.spanName + " was not found", actualSpan, notNullValue());
     assertAttributes(actualSpan, expectedNode);
     assertException(actualSpan, expectedNode);
+    assertThat(actualSpan.getStartEpochSpanNanos(), is(lessThan(actualSpan.getEndSpanEpochNanos())));
```

Comment:
I'd add a error message so that we know which span is failing:
`assertThat("Expected span: " + expectedNode.spanName + " has incorrect start or end time", actualSpan.getStartEpochSpanNanos(), is(lessThan(actualSpan.getEndSpanEpochNanos())));`